src/main/java/org/ibp/api/java/user/UserService.java

```diff
@@ -20,4 +20,13 @@
 	UserDetailDto getUserWithAuthorities(final String cropName, final String programUuid);
 
 	void updateUserProfile(final UserProfileUpdateRequestDTO userProfileUpdateRequestDTO, final WorkbenchUser workbenchUser);
+
+	/**
+	 * Gets the user by id.
+	 *
+	 * @param userId - the user id to match
+	 * @return the user matching the given id
+	 */
+	WorkbenchUser getUserById(final Integer userId);
```

Comment:
No not expose WorkbenchUser in this service, use UserDetailDto

Reply:
done.